tests/testthat/test-summary.pop_data.R

```diff
@@ -14,23 +14,10 @@ test_that("`summary.pop_data()` produces an error when wrong stratification is p
   )
 })
 
-test_that("`summary.pop_data()` does not produce an error when NULL", {
```

Comment:
`expect_snapshot()` will alert us if an unexpected error occurs, so no need to explicitly check for errors anymore.